development/nautobot_config.py

```diff
@@ -133,6 +133,12 @@ def _get_bool_env(name: str):
 # Each key in the dictionary is the name of an installed plugin and its value is a dictionary of settings.
 PLUGINS_CONFIG = {
     "nautobot_chatops": {
+        "ansible": {
+            "tower_uri": os.getenv("NAUTOBOT_TOWER_URI"),
+            "tower_username": os.getenv("NAUTOBOT_TOWER_USERNAME"),
+            "tower_password": os.getenv("NAUTOBOT_TOWER_PASSWORD"),
+            "tower_verify_ssl": _get_bool_env("NAUTOBOT_TOWER_VERIFY_SSL", True),
+        },
```

Comment:
Could we remove the nesting here?
```suggestion
        "ansible_tower_uri": os.getenv("NAUTOBOT_TOWER_URI"),
        "ansible_tower_username": os.getenv("NAUTOBOT_TOWER_USERNAME"),
        "ansible_tower_password": os.getenv("NAUTOBOT_TOWER_PASSWORD"),
        "ansible_tower_verify_ssl": _get_bool_env("NAUTOBOT_TOWER_VERIFY_SSL", True),
```

Reply:
Fixed.

Reply:
While we're at it, can we make the OS environment variable more descriptive? `NAUTOBOT_CHATOPS_ANSIBLE_TOWER_URI`? I know it's long but can we cut down on overlap?

Reply:
Unifying environment variable names is a good point. I already changed some of it (e.g. `ENABLE_MATTERMOST` => `NAUTOBOT_CHATOPS_ENABLE_MATTERMOS`), and it brought some confusion to @pke11y . Will keep track of this issue for the later time.